### PR TITLE
Trying to fix deployments tests, added Creaper

### PR DESCRIPTION
**Pull request by honza-kasik**
- Added Creaper
- Possibly fixed situations when reload required windows appeared
- Removed couple of unnecessary `Library.letsSleep()`

### CODE REVIEW

File: basic/src/test/java/org/jboss/hal/testsuite/test/runtime/deployments/DeploymentsOperations.java [hunk 1]
```diff
@@ -1,46 +1,74 @@
...
+    private OnlineManagementClient client;
+    private Operations operations;
 
-    public DeploymentsOperations(CliClient client) {
+    public DeploymentsOperations(OnlineManagementClient client) {
         this.client = client;
+        this.operations = new Operations(client);
+    }
+
+    public void verifyExists(String deploymentName) throws Exception {
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/runtime/deployments/DeploymentsOperations.java]: I suggest more explicit method name, e.g. `verifyDeploymentExists`.
WDYT?

File: basic/src/test/java/org/jboss/hal/testsuite/test/runtime/deployments/DeploymentsOperations.java [hunk 2]
```diff
@@ -1,46 +1,74 @@
...
+        String message = String.format("Deployment '%s' should exist!", deploymentName);
+        new ResourceVerifier(Address.of("deployment", deploymentName), client, 10000).verifyExists(message);
+    }
+
+    public void verifyDoesNotExist(String deploymentName) throws Exception {
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/runtime/deployments/DeploymentsOperations.java]: I suggest more explicit method name, e.g. `verifyDeploymentDoesNotExist`.
WDYT?

File: basic/src/test/java/org/jboss/hal/testsuite/test/runtime/deployments/DeploymentsOperations.java [hunk 3]
```diff
@@ -1,46 +1,74 @@
...
+        String message = String.format("Deployment '%s' should not exist!", deploymentName);
+        new ResourceVerifier(Address.of("deployment", deploymentName), client, 10000).verifyDoesNotExist(message);
+    }
+
+    public void undeploy(String deploymentName) throws IOException {
+        operations.invoke(String.format(COMMAND_UNDEPLOY, deploymentName), Address.root());
+    }
+
+    private void verifyEnabled(String deploymentName, boolean isEnabled) throws Exception {
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/runtime/deployments/DeploymentsOperations.java]: I suggest more explicit method name, e.g. `verifyDeploymentEnabled`.
WDYT?

File: basic/src/test/java/org/jboss/hal/testsuite/test/runtime/deployments/DeploymentsOperations.java [hunk 4]
```diff
@@ -1,46 +1,74 @@
...
+    }
+
+    public void verifyIsEnabled(String deploymentName) throws Exception {
+        verifyEnabled(deploymentName, true);
     }
 
-    public boolean exists(String deploymentName) {
-        String command = String.format(COMMAND_READ_RESOURCE, deploymentName);
-        return client.executeForSuccess(command);
+    public void verifyIsDisabled(String deploymentName) throws Exception {
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/runtime/deployments/DeploymentsOperations.java]: I suggest more explicit method name, e.g. `verifyDeploymentDisabled`.
WDYT? 
Etc. in the rest of `DeploymentsOperations` methods.

File: basic/src/test/java/org/jboss/hal/testsuite/test/runtime/deployments/DeploymentsOperations.java [hunk 5]
```diff
@@ -1,46 +1,74 @@
...
+    }
+
+    private void verifyEnabled(String deploymentName, boolean isEnabled) throws Exception {
+        String status = isEnabled ? "enabled" : "disabled";
+        Address address = Address.of("deployment", deploymentName);
+        new ResourceVerifier(address, client, 10000).verifyAttribute("enabled", isEnabled,
+                "Deployment should be " + status);
+    }
+
+    public void verifyIsEnabled(String deploymentName) throws Exception {
```
Comment (zebrik) [inline basic/src/test/java/org/jboss/hal/testsuite/test/runtime/deployments/DeploymentsOperations.java]: I suggest more explicit method name, e.g. `verifyDeploymentEnabled`.
WDYT?

**Comment by honza-kasik** [comment]
Including 'deployment' in method names is definitely good idea, refactored.

**Comment by zebrik** [comment]
LGTM now. Thank you!

**Pull request merged**